### DISCUSSION TITLE
绝区零代肝

### DISCUSSION THREAD

**Discussion by Qimen-A** [share121/inter-knot#509] [其他]
啥都能肝 （式舆防卫战尽量肝 看角色练度）（老板可以选择包周或者单选 加联系方式）

普通包月 30R
--每日委托 体力（可指定） 刮刮卡 咖啡馆 录像店

精品包月 60R
--每日委托 体力（可指定） 刮刮卡 咖啡馆 录像店 丽都城募 恶名狩猎 版本活动 角色练度培养 零号空洞（全打满） 

版本包月42天 150R
--每日委托 体力（可指定） 刮刮卡 咖啡馆 录像店 丽都城募 恶名狩猎 版本活动 角色练度培养 零号空洞（全打满） 式舆防卫战 新版本主线 地图探索收集
（能打满的都会尽量全打满）

纯手打 本人账号目前49级
仅支持微信付款 联系加Q2232868415

**Comment by DiamondDiavolo** [comment]
这个委托我接了（bushi）
